src/xinterpreter.cpp

```diff
@@ -291,6 +291,13 @@ namespace xpyt
         result["language_info"]["version"] = PY_VERSION;
         result["language_info"]["mimetype"] = "text/x-python";
         result["language_info"]["file_extension"] = ".py";
+
+        result["help_links"] = nl::json::array();
+        result["help_links"][0] = nl::json::object({
+            {"text", "xeus-python Reference"},
```

Comment:
Should we capitalize Reference?

Reply:
Or Xeus...

Reply:
How about "Xeus-Python Reference"?

Reference is capitalized to stay consistent with `ipykernel`: https://github.com/ipython/ipykernel/blob/92f979eda8280a8898d5235c35df62ac3ee15c79/ipykernel/ipkernel.py#L88-L117

Reply:
Sounds good.

Reply:
Done :+1: